### ISSUE TITLE
use Orientation.js for values of ariaOrientation 

### ISSUE THREAD

**Issue by zepumph** [phetsims/sun#596]
From TODO marked in https://github.com/phetsims/gravity-force-lab-basics/issues/134#issuecomment-632366221:

```

          // TODO: this should be an enumeration, https://github.com/phetsims/gravity-force-lab-basics/issues/134
          ariaOrientation: 'horizontal', // specify orientation, read by assistive technology
```

Now we have Orientation.js. 

@jessegreenberg, how do you feel about using that here? Assigning to both of us for whoever gets to it first. Let me know if you feel differently.

**Comment by zepumph** [comment]
Implemented above. Closing
